katalog/kube-state-metrics/deploy.yml

```diff
@@ -1,41 +1,40 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: kube-state-metrics
   labels:
-    app: kube-state-metrics
+    app.kubernetes.io/name: kube-state-metrics
```

Comment:
Since there is no uniform label convention in the different modules yet, I would just stick to `app: <something>` for now.